### PR TITLE
Fix assembly metadata to fix package verifier warnings

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @ajaybhargavb \ @Tratcher

File: src/Microsoft.AspNetCore.WebSockets.Protocol/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -1,37 +1,13 @@
...
+﻿using System.Reflection;
 using System.Resources;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-// General Information about an assembly is controlled through the following 
-// set of attributes. Change these attribute values to modify the information
-// associated with an assembly.
 [assembly: AssemblyTitle("Microsoft.Net.WebSockets")]
-[assembly: AssemblyDescription("")]
```
Comment (pranavkm) [inline src/Microsoft.AspNetCore.WebSockets.Protocol/Properties/AssemblyInfo.cs]: This one prevented the compiler from using the description field from project.json.

File: src/Microsoft.AspNetCore.WebSockets.Protocol/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -1,37 +1,13 @@
-using System.Reflection;
```
Comment (ajaybhargavb) [inline src/Microsoft.AspNetCore.WebSockets.Protocol/Properties/AssemblyInfo.cs]: Since you are here, add license header?

**Comment by ajaybhargavb** [comment]
:shipit:

File: src/Microsoft.AspNetCore.WebSockets.Protocol/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (pranavkm) [inline src/Microsoft.AspNetCore.WebSockets.Protocol/Properties/AssemblyInfo.cs]: Done.

**Pull request merged**